templates/MMM-NHL.njk

```diff
@@ -1,5 +1,5 @@
 {% if loading %}
-    <div class="dimmed light">{{ "LOADING" | translate }}</div>
```

Comment:
It's not necessary to take a new translation key. According to the translator it should first look in the module translations only if it is not present there it should take the one from the core system. With my current installation (magicmirror develop branch) I don't have the issue you described anymore. I remember though having this issue in older modules where I needed to fix the rendering of HTML symbols like https://github.com/fewieden/MMM-soccer/blob/4e6a6d3280a35a9408f0c15dcdeb570e3904c14d/templates/MMM-soccer.njk#L2

Which version of MM do you use?

Reply:
After having a second thought about it we should use the core translations where possible as they are all translated in so many languages. We should apply the fix described above to render the HTML symbol and remove the key from the module translation files.

However still strange why you were running into this issue and it rendered fine on my side.

Reply:
I'm running latest. 2.14
If you adding {{ "LOADING" | translate }} to anywhere in the template you don't see it say? &hellsip; ?  I see it in this, and Hello World if I add it.  If I do a this.translate("LOADING") it works fine.  If you don't see it, seems like its not a MagicMirror bug, otherwise it is.

Reply:
Took a look at the | safe you added, I did the same and it works correctly.  Feels a little goofy that is needed, I guess you just need to add | safe to any translation where you rely on MM translations... (or all translations...)

Reply:
@dannoh I had another look and realized that I set my config to german when I tested the localized date format for the calendar column. It works for the german language. It only prioritizes the translation from the core over the module translation for English. I will try to make a pull request in the next few days to fix it in the MagicMirror project. But this doesn't stop this change as we decided to rely on the core translation anyway.

In the nunjuck template, you need to add the `safe` filter to render the HTML symbol of ellipsis for security reasons. The `safe` filter is therefore only needed if you try to render HTML. `this.translate` is working because it is outside of the scope of nunjuck rendering engine.